packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -87,6 +89,8 @@ class GoTrueClient {
 
   final AuthFlowType _flowType;
 
+  final _log = Logger('supabase.gotrue');
```

Comment:
We could also change this to `supabase.auth`, since I've noticed the term `gotrue` is more and more replaced by `auth` at supabase.